README.md

```diff
@@ -20,7 +24,7 @@ Since the package isn't yet published, you can install from github:
 ```elixir
 def deps do
   [
-    {:exvault, git: "git@github.com:praekeltfoundation/exvault", sparse: "apps/exvault"},
+    {:exvault, git: "https://github.com/praekeltfoundation/exvault.git", sparse: "apps/exvault"},
```

Comment:
We probably want `{:exvault, github: "praekeltfoundation/exvault", sparse: "apps/exvault/"}` here.

Reply:
Is the extra `/` after `apps/exvault` important?

Reply:
Nope, it's accidental and shouldn't be there. We want `"apps/exvault"` as before.